### ISSUE TITLE
NITDA

### ISSUE THREAD

**Issue by Amaechieze58** [calistus-igwilo/nitda-blockchain-scholarship#192]
this is a wonderful program , am learning a lot .this few days have been seasons of renewal. Thank  Calistus Igwilo

**Comment by Bellomusamuhammad** [comment]
Indeed! This program is a life changing one! We really appreciate their hard work. Especially Calistus Igwilo! May your knowledge benefits more.....!

**Comment by alotazeez** [comment]
NITDA is doing greatly with this JavaScript class.
Many thanks...

**Comment by muhddkd** [comment]
We are very excited with this wonderful opportunity

**Comment by Luqmanadeoti01** [comment]
Good morning am Luqman Adeoti from Iseyin oyo state, the programme is interested and appreciative, but its complex for many of us were not used to computer before, because many things to go on practice is not easy android phone.

**Comment by Luqmanadeoti01** [comment]
Javascript and Golang were complex to understand, pls help us to talk to our states mentors especially oyo state they were not helping to know much about our learning.

**Comment by Ismailtsafe** [comment]
I learning
 more ideas from this programmes

Thanks
NITDA

**Comment by Eniiyanu** [comment]
How can I get added to the Oyo state group?

**Comment by Luqmanadeoti01** [comment]
> How can I get added to the Oyo state group?
> 

Search for oyo state nitda from your telwgram.
